scanner/src/main/kotlin/ScannerWrapperConfig.kt

```diff
@@ -28,18 +28,52 @@ data class ScannerWrapperConfig(
     /**
      * The configuration for the [ScannerMatcher].
      */
-    val matcherConfig: ScannerMatcherConfig
+    val matcherConfig: ScannerMatcherConfig,
+
+    /**
+     * If `true`, scan results of this scanner shall be written to the configured scan storages.
+     */
+    val readFromStorage: Boolean?,
+
+    /**
+     * If `true`, scan results for this scanner shall be read from the configured scan storages.
+     */
+    val writeToStorage: Boolean?
 ) {
     companion object {
-        val EMPTY = ScannerWrapperConfig(ScannerMatcherConfig.EMPTY)
+        val EMPTY = ScannerWrapperConfig(
+            matcherConfig = ScannerMatcherConfig.EMPTY,
+            readFromStorage = null,
+            writeToStorage = null
+        )
+
+        /**
+         * The name of the boolean property to configure if scan results shall be read from the configured scan
+         * storages.
+         */
+        internal const val PROP_READ_FROM_STORAGE = "readFromStorage"
+
+        /**
+         * The name of the boolean property to configure if scan results shall be written to the configured scan
+         * storages.
+         */
+        internal const val PROP_WRITE_TO_STORAGE = "writeToStorage"
+
+        private val properties = listOf(PROP_READ_FROM_STORAGE, PROP_WRITE_TO_STORAGE)
```

Comment:
I would rather use `wrapperProperties` or `configProperties`, but maybe for this small class just `properties` is fine?

Reply:
While I could live with just `properties`, IMO it would be nice if the name told what makes these properties special compared to the other (nested) properties of `ScannerMatcherConfig`.

Reply:
This case BTW is slightly different from `ScannerMatcherConfig.properties` which indeed has *all* properties. But this is just a finding; I could still live with just `properties`.

Reply:
Then I would prefer to keep the name (at least for now).